aiida_vasp/io/chgcar.py

```diff
@@ -0,0 +1,38 @@
+"""Tools for parsing CHGCAR files."""
+
+from aiida_vasp.utils.aiida_utils import get_data_class
+from aiida_vasp.io.parser import BaseFileParser
+
+
+class ChgcarParser(BaseFileParser):
```

Comment:
Please add unit tests for this class in aiida_vasp/io/tests/test_chgcar_io.py

Reply:
Added unit test and a dummy CHGCAR file